custom/rch/models.py

```diff
@@ -0,0 +1,212 @@
+    Mother_ASHA_ID = models.IntegerField(null=True)
+    JSY_Beneficiary = models.CharField(null=True, max_length=255)
+    JSY_Payment_Received = models.CharField(null=True, max_length=255)
+    Delete_Mother = models.CharField(null=True, max_length=255)
+    Entry_Type = models.CharField(null=True, max_length=255)
+    LMP_Date = models.DateTimeField(null=True)
+    EDD_Date = models.DateTimeField(null=True)
+    BloodGroup_Test = models.CharField(null=True, max_length=255)
+    LMP_Registration_Yr = models.PositiveSmallIntegerField(null=True)
+    Last_Pregnancy_Complication = models.CharField(null=True, max_length=255)
+    Last_Pregnancy_outcome = models.CharField(null=True, max_length=255)
+    Expected_delivery_place = models.CharField(null=True, max_length=255)
+    No_of_times_Pregnant = models.PositiveSmallIntegerField(null=True)
+    ANC3 = models.DateTimeField(null=True)
+    ANC3_Pregnancy_Week = models.SmallIntegerField(null=True)
+    ANC3_Weight = models.FloatField(null=True)
+    ANC3_BP_Systolic = models.SmallIntegerField(null=True)
+    ANC3_BP_Distolic = models.SmallIntegerField(null=True)
+    ANC3_Hb_gm = models.FloatField(null=True)
+    TT1 = models.DateTimeField(null=True)
+    Maternal_Death = models.SmallIntegerField(null=True)
+    Death_Reason = models.CharField(null=True, max_length=255)
+    PPMC = models.CharField(null=True, max_length=255)
+    Delivery_date = models.DateTimeField(null=True)
+    Delivery_Place_Name = models.CharField(null=True, max_length=255)
+    Delivery_Conducted_By = models.CharField(null=True, max_length=255)
+    Delivery_Type = models.CharField(null=True, max_length=255)
+    Delivery_Outcomes = models.SmallIntegerField(null=True)
+    Live_Birth = models.SmallIntegerField(null=True)
+    Still_Birth = models.SmallIntegerField(null=True)
+    Discharge_Date = models.DateTimeField(null=True)
+    Delivery_Time = models.TimeField(null=True)
+    Discharge_Time = models.TimeField(null=True)
+    Delivery_Complication = models.CharField(null=True, max_length=255)
+    Infant1_Id = models.BigIntegerField(null=True)
+    Infant1_Name = models.CharField(null=True, max_length=255)
+    Infant1_Baby_Cried = models.CharField(null=True, max_length=255)
+    Infant1_Resucitation_Done = models.CharField(null=True, max_length=255)
+    Infant1_Gender = models.CharField(null=True, max_length=255)
+    Infant1_Weight = models.FloatField(null=True)
+    Infant1_Breast_Feeding = models.CharField(null=True, max_length=255)
+    Infant2_Baby_Cried = models.CharField(null=True, max_length=255)
+    Infant2_Gender = models.CharField(null=True, max_length=255)
+    Subcentre_ID = models.IntegerField(null=True)
+    Village_Name = models.CharField(null=True, max_length=255)
+
+    def save(self, *args, **kwargs):
+        try:
+            if self.Delivery_Time:
+                datetime.datetime.strptime(self.Delivery_Time, '%H:%M:%S')
+        except ValueError:
```

Comment:
This is to take care if Delivery_Time is not in correct format. There was once the API returned "%H:%M" (AS-IS) in this field. So this takes care of it so that save call does not fail. I don't think we want the save to fail hence not validating it but just ignoring it if not in required format. Also read that clean method is not called on save(). Should it be called explicitly then?